views/style.scss

```diff
@@ -1607,4 +1607,22 @@ div.names-container {
     margin-top: 50px;
     order: 3;
   }
+}
+
+.person-title {
+  color: #ababab;
+  white-space: nowrap;
+  font-size: 1.3rem;
```

Comment:
This is causing a spacing issue

<img width="849" alt="image" src="https://user-images.githubusercontent.com/54484616/224568626-e01f873d-9089-4429-bf34-81868462bfba.png">

<img width="48" src="https://user-images.githubusercontent.com/54484616/224568631-0e8e470e-a836-4b92-ac53-13fd6e9d68f4.png">

Reply:
@Awesome-E check it out now. fixed :D